diff --git a/src/main/java/com/premiumminds/sonar/postgres/protobuf/Token.java b/src/main/java/com/premiumminds/sonar/postgres/protobuf/Token.java
index 8b412a0..3ce7d5a 100644
--- a/src/main/java/com/premiumminds/sonar/postgres/protobuf/Token.java
+++ b/src/main/java/com/premiumminds/sonar/postgres/protobuf/Token.java
@@ -3374,7 +3374,7 @@ public enum Token
   /**
    * <code>JSON = 486;</code>
    */
-  public static final int JSON_VALUE = 486;
+  public static final int JSON_ENUM_VALUE = 486;
   /**
    * <code>JSON_ARRAY = 487;</code>
    */
@@ -5159,7 +5159,7 @@ public enum Token
     return com.premiumminds.sonar.postgres.protobuf.PgQuery.getDescriptor().getEnumType(72);
   }
 
-  private static final Token[] VALUES = values();
+  private static final Token[] INTERNAL_VALUES = values();
 
   public static Token valueOf(
       com.google.protobuf.Descriptors.EnumValueDescriptor desc) {
@@ -5170,7 +5170,7 @@ public enum Token
     if (desc.getIndex() == -1) {
       return UNRECOGNIZED;
     }
-    return VALUES[desc.getIndex()];
+    return INTERNAL_VALUES[desc.getIndex()];
   }
 
   private final int value;
